src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,44 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_LENGTH = 10;
+    private static final int NOT_FOUND = -1;
+    private K[] keysArray = (K[]) new Object[MAX_LENGTH];
+    private V[] valuesArray = (V[]) new Object[MAX_LENGTH];
+    private int size = 0;
```

Comment:
```suggestion
    private int size;
```
zero is a default value for int type